src/services/refactors/extractMethod.ts

```diff
@@ -855,6 +855,7 @@ namespace ts.refactor.extractMethod {
             return { body: createBlock(body.statements, /*multLine*/ true), returnValueProperty: undefined };
         }
         let returnValueProperty: string;
+        let ignoreReturns = false;

```

Comment:
This would be simpler to just have `visitor` take a parameter, is there a performance concern?